Vagrantfile

```diff
@@ -115,27 +115,9 @@ Vagrant.configure(2) do |config|
 
   # Centos7 box
   config.vm.define "hoot_centos7", autostart: false do |hoot_centos7|
-    # This seems to be the "latest" version of centos7
-    # hoot_centos7.vm.box = "centos/7"
-    # hoot_centos7.vm.box_url = "https://atlas.hashicorp.com/centos/boxes/7"
-
-    # There are a lot of different Centos7.2 boxes.
-    hoot_centos7.vm.box = "bento/centos-7.2"
-    hoot_centos7.vm.box_url = "https://atlas.hashicorp.com/bento/boxes/centos-7.2"
-
-    # Stop the default vagrant rsyncing
-    config.vm.synced_folder '.', '/home/vagrant/sync', disabled: true
-
-    # NOTE: Networking needs some tweaking.
-    # If you don't want to use NFS, then something like this will work:
-    # hoot_centos7.vm.synced_folder '.', '/home/vagrant/hoot', type: 'rsync'
-
-    hoot_centos7.vm.network "private_network", ip: "192.168.33.10"
-    hoot_centos7.nfs.map_uid = Process.uid
-    hoot_centos7.nfs.map_gid = Process.gid
-    # Enabled async, this makes small file writes much faster (e.g. unzip)
-    hoot_centos7.vm.synced_folder ".", "/home/vagrant/.hoot-nfs", type: "nfs", :linux__nfs_options => ['rw','no_subtree_check','all_squash','async']
-    hoot_centos7.bindfs.bind_folder "/home/vagrant/.hoot-nfs", "/home/vagrant/hoot", perms: nil
```

Comment:
Does removing nfs work for both use cases of running in AWS and running locally via VirtualBox?

Reply:
Yes, because we're now using the `hoot/centos7-minimal` box which has the VirtualBox guest additions installed unlike the previously used box, `bento/centos-7.2`.  NFS was always disabled on AWS via an override and as far as I can tell the EC2 instances wouldn't have access to the hardcoded server IP (`192.168.33.10`).

Reply:
That is great that we have a hoot box.  Does it have swap space?

Reply:
The VirtualBox provided versions use a "stock" LVM configuration including swap space, the AWS AMI does not.  This is because AMIs can have their EBS-backed disk expanded to the desired size (`cloud-init` grows partition on first boot) and AWS recommends against having swap as it can easily lead to IOP spikes that have pricing consequences.